client/src/components/Workbench/WorkbenchSidebar/WorkbenchSidebar.test.js

```diff
@@ -3,10 +3,10 @@ import { MemoryRouter, Route } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { render } from '@testing-library/react';
 import configureStore from 'redux-mock-store';
-import { initialState as workbench } from '../../redux/reducers/workbench.reducers';
-import { initialState as notifications } from '../../redux/reducers/notifications.reducers';
-import { initialTicketCreateState as ticketCreate } from '../../redux/reducers/tickets.reducers';
-import Sidebar from './index';
+import { initialState as workbench } from 'redux/reducers/workbench.reducers';
+import { initialState as notifications } from 'redux/reducers/notifications.reducers';
+import { initialTicketCreateState as ticketCreate } from 'redux/reducers/tickets.reducers';
```

Comment:
I defer to y'all, but I am curious.

Relative links are clear. These links are vague. As one unfamiliar with the code, I might come across these imports and ask more questions than I think I should have about imports.

<details><summary>Questions</summary>

1. Are these coming from the `redux` package?
2. Oh, slashes and words that match our work, so these are from our code?
3. What is the base path?

</details>

Reply:
Yes these are from our code. I used the autocomplete in vscode to complete the import and this is the path it populated. I can switch this to a relative path which would be clearer and since that's how other components do imports as well.

Reply:
🤷‍♂️  I'm leaving my opinion out of code I don't need to worry about.